scripts/get-user.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+"""Outputs details of a random active unlocked user with a given role.
+
+For supplier users, providing additional <framework> and <lot> arguments allows
+filtering by suppliers who have published services on the given framewok and lot.
```

Comment:
This implies the framework/lot args are optional. If you ask for a supplier, the script dies if you don't provide at least a framework.